pinot-common/src/main/java/org/apache/pinot/common/utils/FileUploadDownloadClient.java

```diff
@@ -1004,6 +1004,25 @@ public int downloadFile(URI uri, File dest, AuthProvider authProvider, List<Head
     return _httpClient.downloadFile(uri, HttpClient.DEFAULT_SOCKET_TIMEOUT_MS, dest, authProvider, httpHeaders);
   }
 
+  /**
+   * Download and untar a file.
```

Comment:
(nit) Please add few more sentences about this method

Reply:
done